app/shared/api-services/poll/answer-factory.tests.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+      answer,
+      answerFactory,
+      scope,
+      pollId;
+
+  beforeEach(module('apiServices.answer.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Answer_) {
+    answerFactory = _Answer_;
+    $httpBackend = _$httpBackend_;
+
+    scope = {
```

Comment:
In this case, especially since the tests are treating this data differently, set the scope per test instead of in this `beforeEach`. It will make the tests more readable at a glance.